### PR TITLE
netcoreapp2.0

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: test/MusicStore.E2ETests/SmokeTests.cs [hunk 1]
```diff
@@ -22,36 +22,34 @@ public SmokeTests_X86(ITestOutputHelper output)
...
-        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Portable)]
-        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Standalone)]
-        [InlineData(ServerType.Kestrel, RuntimeFlavor.Clr, RuntimeArchitecture.x86, ApplicationType.Portable)]
-        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Portable)]
-        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Standalone)]
-        [InlineData(ServerType.IISExpress, RuntimeFlavor.Clr, RuntimeArchitecture.x86, ApplicationType.Portable)]
-        [InlineData(ServerType.IISExpress, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Portable)]
-        [InlineData(ServerType.IISExpress, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Standalone)]
+        [InlineData(ServerType.WebListener, RuntimeArchitecture.x86, ApplicationType.Portable)]
+        [InlineData(ServerType.WebListener, RuntimeArchitecture.x86, ApplicationType.Portable)]
```
Comment (mikeharder) [inline test/MusicStore.E2ETests/SmokeTests.cs]: Duplicate line

File: test/MusicStore.E2ETests/SmokeTests.cs [hunk 2]
```diff
@@ -22,36 +22,34 @@ public SmokeTests_X86(ITestOutputHelper output)
...
+        [InlineData(ServerType.WebListener, RuntimeArchitecture.x86, ApplicationType.Standalone)]
+        [InlineData(ServerType.Kestrel, RuntimeArchitecture.x86, ApplicationType.Portable)]
+        [InlineData(ServerType.Kestrel, RuntimeArchitecture.x86, ApplicationType.Portable)]
```
Comment (mikeharder) [inline test/MusicStore.E2ETests/SmokeTests.cs]: Duplicate line

File: test/MusicStore.E2ETests/SmokeTests.cs [hunk 3]
```diff
@@ -22,36 +22,34 @@ public SmokeTests_X86(ITestOutputHelper output)
...
+        [InlineData(ServerType.Kestrel, RuntimeArchitecture.x86, ApplicationType.Standalone)]
+        [InlineData(ServerType.IISExpress, RuntimeArchitecture.x86, ApplicationType.Portable)]
+        [InlineData(ServerType.IISExpress, RuntimeArchitecture.x86, ApplicationType.Portable)]
```
Comment (mikeharder) [inline test/MusicStore.E2ETests/SmokeTests.cs]: Duplicate line

File: test/MusicStore.E2ETests/SmokeTests.cs [hunk 4]
```diff
@@ -67,37 +65,35 @@ public SmokeTests_X64(ITestOutputHelper output)
...
-        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
-        [InlineData(ServerType.Kestrel, RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
-        [InlineData(ServerType.IISExpress, RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        [InlineData(ServerType.IISExpress, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        [InlineData(ServerType.IISExpress, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(ServerType.WebListener, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        [InlineData(ServerType.WebListener, RuntimeArchitecture.x64, ApplicationType.Portable)]
```
Comment (mikeharder) [inline test/MusicStore.E2ETests/SmokeTests.cs]: Dup

File: test/MusicStore.E2ETests/SmokeTests.cs [hunk 5]
```diff
@@ -67,37 +65,35 @@ public SmokeTests_X64(ITestOutputHelper output)
...
+        [InlineData(ServerType.WebListener, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(ServerType.Kestrel, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        [InlineData(ServerType.Kestrel, RuntimeArchitecture.x64, ApplicationType.Portable)]
```
Comment (mikeharder) [inline test/MusicStore.E2ETests/SmokeTests.cs]: Dup

File: test/MusicStore.E2ETests/SmokeTests.cs [hunk 6]
```diff
@@ -67,37 +65,35 @@ public SmokeTests_X64(ITestOutputHelper output)
...
+        [InlineData(ServerType.Kestrel, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(ServerType.IISExpress, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        [InlineData(ServerType.IISExpress, RuntimeArchitecture.x64, ApplicationType.Portable)]
```
Comment (mikeharder) [inline test/MusicStore.E2ETests/SmokeTests.cs]: Dup

**Comment by pakrym** [comment]
:up::date:

**Pull request merged**